core/src/main/java/com/sequenceiq/cloudbreak/job/StackStatusCheckerJob.java

```diff
@@ -166,6 +178,27 @@ protected void executeTracedJob(JobExecutionContext context) throws JobExecution
         }
     }
 
+    private boolean shouldSkipStatusCheck() {
+        if (flowLogService.isOtherFlowRunning(getStackId())) {
+            LOGGER.debug("StackStatusCheckerJob cannot run, because flow is running for stack: {}", getStackId());
+            return true;
+        }
+        Optional<FlowLog> lastFlowLog = flowLogService.getLastFlowLogWithEndTime(getStackId());
+        if (lastFlowLog.isPresent()) {
+            FlowLog flowLog = lastFlowLog.get();
+            Instant skipTimeInstant = clock.nowMinus(Duration.ofMinutes(skipWindow));
+            Instant lastFlowLogEndTimeInstant = Instant.ofEpochMilli(flowLog.getEndTime());
+            if (lastFlowLogEndTimeInstant.isAfter(skipTimeInstant)) {
+                LOGGER.debug("Last flow log endtime in UTC: {}", lastFlowLogEndTimeInstant.atZone(ZoneOffset.UTC));
```

Comment:
Do we really need these log messages that only contain a few UTC timestamps? The last log message + skipWindow seems enough.

Reply:
I added these for a debugging reason to see if the calculation is wrong.

Reply:
Ok, but hopefully you have tested the calculation logic and everything worked well and you can remove these :)